2216/2/versus.rb

```diff
@@ -0,0 +1,101 @@
+require 'optparse'
+require 'russian_obscenity'
+require 'unicode'
+require 'active_support'
+require 'active_support/core_ext'
+require_relative 'bad_words_analyzer'
+require_relative 'top_words_analyzer'
+require_relative 'battle'
+require_relative 'bad_words_table'
+require_relative 'rapper'
+
+# get env variables
+# --------------------------------------------------------------------------------------------
+options = { top_bad_words: nil, name: nil, top_words: nil }
+
+parser = OptionParser.new do |opts|
+  opts.banner = 'Usage: versus.rb [options]'
+  opts.on('--top-bad-words number') do |number|
+    options[:top_bad_words] = number
+  end
+
+  opts.on('--name name') do |name|
+    options[:name] = name
+  end
+
+  opts.on('--top-words number') do |number|
+    options[:top_words] = number
+  end
+end
+
+parser.parse!
+# ---------------------------------------------------------------------------------------------
+
+if options.values.all?(&:nil?)
```

Comment:
The code should not live here.
The only thing you should do in your `versus.rb` file is to parse parameters with `optparse` and then pass them to the corresponding class for further processing!